### DISCUSSION TITLE
How to connect SQL events in Wazuh

### DISCUSSION THREAD

**Discussion by vre344** [wazuh/wazuh#16875] [Q&A]
Good afternoon! I can't figure out how to get events from SQL to Wazuh. Can you explain how to set it up correctly? 
In the future, I would connect to the Kaspersky Security Center SQL database, as if using Syslog, Kaspersky needs an extended license

**Comment by mdiego92** [comment]
Hello @vre344,

If you have a log source in your Wazuh Manager environment, you could use logcollector to monitor the file/s:
https://documentation.wazuh.com/current/user-manual/reference/ossec-conf/localfile.html?highlight=localfile

The other possibility would be receiving syslogs logs:

- [From a specific port](https://documentation.wazuh.com/current/user-manual/capabilities/log-data-collection/how-it-works.html#receiving-syslog-logs-in-a-custom-port)
- [Storing syslog logs in a plaintext file and monitoring it with Wazuh](https://documentation.wazuh.com/current/user-manual/capabilities/log-data-collection/how-it-works.html)

Please let me know if you have any remaining questions!

**Reply by vre344** [reply]
Thank you!
